mmd_tools/operators/morph.py

```diff
@@ -239,13 +240,13 @@ def execute(self, context):
             return { 'FINISHED' }
         mat_data = morph.data[morph.active_material_data]
 
-        if mmd_root.advanced_mode:
-            relMesh = rig.findMesh(mat_data.related_mesh)
-            if relMesh is not None:
-                meshObj = relMesh
-            else:
-                self.report({ 'ERROR' }, "The related mesh can't be found")
-                return { 'CANCELLED' }
+        # if mmd_root.advanced_mode:
+        relMesh = rig.findMesh(mat_data.related_mesh)
+        if relMesh is not None:
+            meshObj = relMesh
+        else:
+            self.report({ 'ERROR' }, "The related mesh can't be found")
+            return { 'CANCELLED' }
         
```

Comment:
Actually, you can refine the code from this:
>
relMesh = rig.findMesh(mat_data.related_mesh)
if relMesh is not None:
            meshObj = relMesh
 else:
            self.report({ 'ERROR' }, "The related mesh can't be found")
            return { 'CANCELLED' }

to this:
>
meshObj = rig.findMesh(mat_data.related_mesh)

And other simliar places as well. :)